CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0] - 2021-08-20
```

Comment:
:mega: We should only add this section just before releasing. These changes should remain under `Unreleased` for now.

```suggestion

```